### ISSUE TITLE
Add config/production/*.php files to containers

### ISSUE THREAD

**Issue by bytestream** [supportpal/helpdesk-install#60]
We need the ability to add `config/production/*.php` files to the containers.  For example:
* https://docs.supportpal.com/current/SAML+Authentication
* https://docs.supportpal.com/current/Updating+Config+Files

It's possible at the moment by adding a volume definition to the docker compose yml files:
```yml
- ./saml.php:/supportpal/config/production/saml.php
```

However this is troublesome. If you remove the volume from the yml file, an empty `config/production/saml.php` sticks around. That causes a PHP error because SupportPal loads all `config/production/*.php` files and expects them to return arrays.  I believe the file sticks around because it conflicts with the existing external volume definition:
```
- supportpal_config:/supportpal/config/production
```

**Comment by Gaitholabi** [comment]
I think that the simplest solution here is issuing a 
`docker cp file container:/path`

Where `/path` is inside a mounted volume so the file would persist

**Comment by bytestream** [comment]
I've updated docs.supportpal.com and the changes will be available in when v3.7.0 is published.
